modules/travel_pay/spec/services/service_spec.rb

```diff
@@ -52,8 +62,8 @@
     end
 
     it 'returns sorted and parsed claims' do
```

Comment:
I'm fine with removing sorting, but if we're doing that, we should:
* remove anything to do with IDs, they were only there to ensure ordering
* expect title-cased statuses in a non-ordered way